src/Client.js

```diff
@@ -44,4 +44,12 @@ export default class Client {
       { email, password },
     )).data;
   }
+
+  async forgotPassword(email) {
+    const { protocol, hostname, port } = this.options;
+    return (await axios.post(
```

Comment:
The same about a `response` variable to separate from putting all together on return statement.